staging/kubefed/templates/install-crds.yaml

```diff
@@ -0,0 +1,86 @@
+{{ if or (not .Values.global.scope) (eq .Values.global.scope "Cluster") }}
+{{if not (.Capabilities.APIVersions.Has "types.kubefed.io/v1beta1") }}
```

Comment:
Here, too, we don't want to test for the capability

Reply:
? are the end closures correct? he should check the first time if installed or not, and the upgrade path should always happen. I think this is ok?